tests/test-sources/modules/output.nix

```
@@ -99,4 +99,54 @@
         }
       ];
     };
+
+  with-providers = {
+    withNodeJs = true;
+    withPerl = true;
+    withPython3 = true;
+    withRuby = true;
+
+    extraConfigLua = ''
+      if vim.fn.executable("nvim-node") != 1 then
```

```
error parsing: error occurred while tokenizing: unexpected character !
```

Looks like I got my lua operators wrong:

```suggestion
      if vim.fn.executable("nvim-node") ~= 1 then
```